internal/pkg/services/object-storage/utils/utils.go

```diff
@@ -30,3 +31,26 @@ func GetCredentialsGroupName(ctx context.Context, apiClient ObjectStorageClient,
 
 	return "", fmt.Errorf("could not find Object Storage credentials group name")
 }
+
+func GetCredentialsName(ctx context.Context, apiClient ObjectStorageClient, projectId, credentialsGroupId, keyId string) (string, error) {
+	req := apiClient.ListAccessKeys(ctx, projectId)
+	req = req.CredentialsGroup(credentialsGroupId)
+	resp, err := req.Execute()
+
+	if err != nil {
+		return "", fmt.Errorf("list Object Storage credentials: %w", err)
+	}
+
+	credentials := resp.AccessKeys
+	if credentials == nil {
+		return "", fmt.Errorf("nil Object Storage credentials list: %w", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("nil Object Storage credentials list)
```

Reply:
In here we don't need to wrap the `err` because that error is from another operation and is already checked above